package.json

```
@@ -11,7 +11,7 @@
 		"url": "https://sindresorhus.com"
 	},
 	"engines": {
-		"node": ">=8"
+		"node": "^10.17"
```

This is also incorrect.

```suggestion
		"node": ">=10.17"
```